packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +75,26 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+
+    if (this.props.srcdoc) {
+      if (frameOptions.iframeAttrs === null
+        || frameOptions.iframeAttrs === undefined) {
+        frameOptions.iframeAttrs = {};
+      }
```

Comment:
```suggestion
   frameOptions.iframeAttrs = this.props.srcdoc && iframeOptions.attrs ? frameOptions.iframeAttrs : {}
```

Think this could be simplified this way.

Reply:
I'll re-work this logic a bit, but I get your point. We don't need to include `this.props.srcdoc` in the check.